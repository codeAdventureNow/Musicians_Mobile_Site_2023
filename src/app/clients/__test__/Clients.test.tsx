import Clients from '../page';
import { render } from '@testing-library/react';

describe('Clients', () => {
  it('should render the "Clients" page', () => {
    const { asFragment } = render(<Clients />);

    expect(asFragment()).toMatchSnapshot();
  });
});
