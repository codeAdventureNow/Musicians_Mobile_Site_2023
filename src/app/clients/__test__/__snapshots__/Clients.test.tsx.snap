// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clients should render the "Clients" page 1`] = `
<DocumentFragment>
  <div
    class="homepage"
  >
    <h2
      class="paragraph"
    >
      Clients page
    </h2>
    <section
      class="section"
    >
      <div
        class="container"
      >
         
        <p>
          Loading...
        </p>
      </div>
    </section>
  </div>
</DocumentFragment>
`;
